### PR TITLE
Chore: add helm

### PR DESCRIPTION
**Pull request by MrPink**
Welcome to Helm! More information about Helm can be found here https://helm.sh/

**These will most likely evolve in time and come with fewer defaults**

* This allows us to package Aether Kubernetes resources as Helm packages which removes the complexity of distributing these resources in an organised and efficient manner. This is comparable to Debian's APT package system

* We host these Charts on http://charts.eha.im:8080/api/charts 
* The Helm charts use an override YAML where we can specify things to specific to the environment

**TO DO**
* Helm release and deployment workflow using Travis CI

### CODE REVIEW

File: helm/kernel/values.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+release: 0.5.1
+service:
+  name: nginx
+  type: NodePort
+  externalPort: 80
+  internalPort: 80
+environment: "dev"
+ums_url: ums-dev
+domain: gather2.org
+trusted_origins: .ehealthafrica.org
```
Comment (quesebifurcan) [inline helm/kernel/values.yaml]: Not sure if `.ehealthafrica.org` is enough -- we might need `.gather2.org` instead/as well. Let's test and see.

File: helm/odk/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  http://{{ . }}
+{{- end }}
+{{- else if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "odk.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get svc -w {{ template "odk.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "odk.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```
Comment (quesebifurcan) [inline helm/odk/templates/NOTES.txt:13]: Does the helm linter catch jsonpath lookup errors?

File: helm/odk/templates/db-init-config-map.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+kind: ConfigMap
+metadata:
+  name: odk-db-init 
+data:
+  db-init.sh: |
+    #!/bin/bash
+
+    psql -c "CREATE DATABASE $RDS_DB_NAME;"
+    psql -c "CREATE USER $RDS_USERNAME WITH PASSWORD '$RDS_PASSWORD';"
+    psql -c "GRANT ALL PRIVILEGES ON DATABASE $RDS_DB_NAME TO $RDS_USERNAME;"
```
Comment (quesebifurcan) [inline helm/odk/templates/db-init-config-map.yaml]: Should we run bash scripts with some options perhaps?
`set -e` or `set -Eeuo(x) pipefail` (depending on the context)?

**Review by quesebifurcan — CHANGES_REQUESTED** [review]

File: helm/odk/templates/db-init-config-map.yaml (hunk 3 above)
Comment (MrPink) [reply helm/odk/templates/db-init-config-map.yaml]: well caught, I think `set -e` suffices

File: helm/odk/templates/NOTES.txt (hunk 2 above)
Comment (MrPink) [reply helm/odk/templates/NOTES.txt:13]: this outside the boundary of Helm's duties

**Review by quesebifurcan — DISMISSED** [review]
Great! Looks good to me

**Review by quesebifurcan — DISMISSED** [review]

**Review by quesebifurcan — APPROVED** [review]

**Pull request merged**